sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubListener.cs

```diff
@@ -71,6 +71,7 @@ void IListener.Cancel()
 
         void IDisposable.Dispose()
         {
+            StopAsync(CancellationToken.None).Wait();
```

Comment:
Should we consider `GetAwaiter().GetResult()` here to avoid the `AggregateException` wrapper on failure?